src/profile-logic/gecko-profile-versioning.js

```diff
@@ -332,6 +332,27 @@ const _upgraders = {
     convertToVersionTenRecursive(profile);
   },
   [11]: profile => {
+    // Ensure there is always a pid in the profile meta AND upgrade
```

Comment:
I took your advice on adding this as a gecko profile upgrade. Could you take a look at it again @canaltinova ? I originally didn't think to do this because I was worried about some pids being missed, but thinking through it, the upgraders should catch everything, since this change happened really far in the past.